packages/electron-builder-squirrel-windows/src/SquirrelWindowsTarget.ts

```diff
@@ -56,6 +89,16 @@ export default class SquirrelWindowsTarget extends Target {
       packager: this.packager,
     })
 
+    if (this.options.msi) {
+      await packager.info.callArtifactBuildCompleted({
+        file: artifactPath.replace(".exe", ".msi"),
+        target: this,
+        arch,
+        safeArtifactName: `${sanitizedName}-Setup-${version}${getArchSuffix(arch)}.msi`,
```

Comment:
Nit: `safeArtifactName` is the same format. Could be worth pulling out into a common function
```
const safeArtifactName = (ext) => `${sanitizedName}-Setup-${version}${getArchSuffix(arch)}.${ext}`
```

Reply:
Fixed.